wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/OAuthUseCase.swift

```diff
@@ -109,7 +109,10 @@ public class OAuthUseCase: OAuthUseCaseInterface {
 
     @MainActor
     private func execute(authorizationRequest: OIDAuthorizationRequest) async throws -> OAuthResponse {
-        guard let userAgent = OIDExternalUserAgentIOS(presenting: rootViewController) else {
+        guard let userAgent = OIDExternalUserAgentIOS(
+            presenting: rootViewController,
+            prefersEphemeralSession: true)
+        else {
```

Comment:
💅 Somehow I would rather format this way, but it's up to you:
```suggestion
        guard let userAgent = OIDExternalUserAgentIOS(
            presenting: rootViewController,
            prefersEphemeralSession: true
        ) else {
```